styles/php85.css

```
@@ -0,0 +1,4002 @@
+    display: flex
+}
+
+:where(.code-container .header>:not(:last-child)) {
+    --tw-space-x-reverse: 0;
+    margin-inline-start: calc(calc(var(--spacing) * 2) * var(--tw-space-x-reverse));
+    margin-inline-end: calc(calc(var(--spacing) * 2) * calc(1 - var(--tw-space-x-reverse)))
+}
+
+.code-container .header {
+    border-bottom-style: var(--tw-border-style);
+    border-bottom-width: 1px;
+    border-color: var(--color-gray-200);
+    padding-inline: calc(var(--spacing) * 3);
+    padding-block: calc(var(--spacing) * 2);
+    font-size: var(--text-sm);
+    line-height: var(--tw-leading, var(--text-sm--line-height));
+    --tw-font-weight: var(--font-weight-semibold);
+    font-weight: var(--font-weight-semibold)
+}
+
+.code-container .header:where(.dark,.dark *) {
+    border-color: #ffffff0d
+}
+
+@supports (color:color-mix(in lab, red, red)) {
+    .code-container .header:where(.dark,.dark *) {
+        border-color: color-mix(in oklab, var(--color-white) 5%, transparent)
+    }
+}
+
+.code-container .header div {
+    align-items: center;
+    display: flex
+}
+
+:where(.code-container .header div>:not(:last-child)) {
+    --tw-space-x-reverse: 0;
+    margin-inline-start: calc(calc(var(--spacing) * 3) * var(--tw-space-x-reverse));
+    margin-inline-end: calc(calc(var(--spacing) * 3) * calc(1 - var(--tw-space-x-reverse)))
+}
+
+.code-container .header a {
+    font-size: var(--text-xs);
+    line-height: var(--tw-leading, var(--text-xs--line-height));
+    color: var(--color-gray-600);
+    text-decoration-line: underline
+}
+
+@media (hover: hover) {
+    .code-container .header a:hover {
+        text-decoration-line: none
+    }
+}
+
+.code-container .header a:where(.dark,.dark *) {
+    color: var(--color-gray-400)
+}
+
+.code-container button {
+    cursor: pointer;
+    border-radius: var(--radius-lg);
+    padding: calc(var(--spacing) * 2)
+}
+
+@media (hover: hover) {
+    .code-container button:hover {
+        background-color: var(--color-gray-100)
+    }
+
+    .code-container button:where(.dark,.dark *):hover {
+        background-color: #ffffff0d
+    }
+
+    @supports (color:color-mix(in lab, red, red)) {
+        .code-container button:where(.dark,.dark *):hover {
+            background-color: color-mix(in oklab, var(--color-white) 5%, transparent)
+        }
+    }
+}
+
+.code-container button svg {
+    width: calc(var(--spacing) * 4);
+    height: calc(var(--spacing) * 4)
+}
+
+.code-container pre {
+    min-height: 100%;
+    font-size: var(--text-xs);
+    line-height: var(--tw-leading, var(--text-xs--line-height));
+    --tw-leading: var(--leading-loose);
+    line-height: var(--leading-loose);
+    white-space: pre-wrap;
+    display: flex
+}
+
+@media (min-width: 40rem) {
+    .code-container pre {
+        font-size: var(--text-sm);
+        line-height: var(--tw-leading, var(--text-sm--line-height))
+    }
+}
+
+.code-container code {
+    height: 100%;
+    padding: calc(var(--spacing) * 4);
```

```suggestion
    padding: var(--spacing);
```
Currently there's too much padding since Prism.js is also adding padding and margin on the `<pre>` element.